tests/test_rdm.py

```diff
@@ -289,6 +289,54 @@ def test_positive_transform(self):
         self.assertEqual(pos_rdm.n_cond, rdms.n_cond)
         assert np.all(pos_rdm.dissimilarities >= 0)
 
+    def test_minmax_transform(self):
+        from rsatoolbox.rdm import minmax_transform
+        dis = np.zeros((8, 10))
+        mes = "Euclidean"
+        des = {'subj': 0}
+        pattern_des = {'type': np.array([0, 1, 2, 2, 4])}
+        rdm_des = {'session': np.array([0, 1, 2, 2, 4, 5, 6, 7])}
+        rdms = rsr.RDMs(dissimilarities=dis,
+                        rdm_descriptors=rdm_des,
+                        pattern_descriptors=pattern_des,
+                        dissimilarity_measure=mes,
+                        descriptors=des)
+        mm_rdm = minmax_transform(rdms)
+        self.assertEqual(mm_rdm.n_rdm, rdms.n_rdm)
```

Comment:
I don't think these tests for the number of rdms and conditions add much here. My suggestions would be to remove them.

Reply:
Hi Jasper, thanks for the review! I see that the sqrt_transform and rank_transform has a similar test (which was what I based my test upon). My thinking is that it can still catch some issues, which without it, cannot find -- e.g. that calling the functions will not throw an error. I fully agree that, with more time, actually adding a test element regarding the values itself would be more helpful.